rdtools/normalization.py

```diff
@@ -76,17 +77,17 @@ def pvwatts_dc_power(poa_global, P_ref, T_cell=None, G_ref=1000, T_ref=25,
     ----------
     poa_global : pd.Series
         Total effective plane of array irradiance.
-    P_ref : float
+    pmp_ref : float
```

Comment:
`pmp_ref` is the closest term I found to mean array nameplate capacity.  Possible that I missed a better alternative.

Reply:
`power_dc_rated` ?

Reply:
pmp_ref works for me here.

Reply:
looks like T_cell, G_ref and T_ref could be changed though?  'temperature_cell', 'poa_ref' and 'temperature_ref'?

Reply:
Let's stick with `power_dc_rated`. Change `irradiance_ref` to `poa_global_ref`, and `temperature_ref` to `temperature_cell_ref`

Reply:
@cwhanse @wholmgren I see that `gamma_pdc` is used in pvlib, but is listed as deprecated in pv-terms. Is there a plan to change to `gamma_pmp_normalized` in pvlib?